src/block/block_open.c

```diff
@@ -245,6 +245,18 @@ __wt_block_open(WT_SESSION_IMPL *session, const char *filename, uint32_t objecti
     }
     WT_ERR(__wt_open(session, filename, WT_FS_OPEN_FILE_TYPE_DATA, flags, &block->fh));
 
+#ifndef _MSC_VER
+    /*
+     * We need to do this as close to __wt_open as possible as there is a descriptor block read
+     * further down which requires the extent lists to be initialized.
+     */
+    if (extent_str != NULL)
```

Comment:
I think I can remove the NULL check here.

Reply:
[4b58531](https://github.com/wiredtiger/wiredtiger/pull/11505/commits/4b585318288245f8b3102f83039d37f9d7b0fea1)